parley/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "parley"
+version = "0.1.0"
+description = "Parley provides an API for implementing rich text layout."
```

Comment:
Not really relevant for this PR, but I think the only useful part of this description is `Rich text layout`.

Reply:
Sounds like a good PR in the future! :)